smartgymapi/handlers/busyness.py

```diff
@@ -0,0 +1,212 @@
+import logging
+import requests
+
+from datetime import date, datetime, time, timedelta
+from itertools import groupby
+from pytz import timezone
+
+from marshmallow import ValidationError
+
+from pyramid.httpexceptions import HTTPBadRequest
+from pyramid.view import view_config, view_defaults
+
+from smartgymapi.models.gym import get_gym
+
+from smartgymapi.lib.factories.busyness import BusynessFactory
+from smartgymapi.lib.validation.busyness import BusynessSchema
+
+log = logging.getLogger(__name__)
+
+
+@view_defaults(containment=BusynessFactory,
+               permission='busyness',
+               renderer='json')
+class RESTBusyness(object):
+
+    def __init__(self, request):
+        self.request = request
+        self.hour_count = {}
+        self.settings = request.registry.settings
+
+    @view_config(name='past', context=BusynessFactory,
+                 request_method="GET")
```

Comment:
je `request_method` kan in `view_defaults` al op "GET" aangezien zo te zien alles GET is.